src/generic_core/uproxy_core.ts

```diff
@@ -636,8 +636,14 @@ export class uProxyCore implements uproxy_core_api.CoreApi {
       //       that's extremely verbose right now.
       ui.update(uproxy_core_api.Update.CLOUD_INSTALL_STATUS, 'Installing...');
 
+      // Attempt to install.  If install fails, retry will attempt again
+      // up to MAX_INSTALLS times.  Failure may occur because we have just
+      // created the server and it is not yet ready for SSH.
       // TODO: The provisioning module should return the username!
-      return installer.install(host, port, 'root', serverInfo.ssh.private);
+      var installFunc = installer.install.bind(
+          installer, host, port, 'root', serverInfo.ssh.private);
+      var MAX_INSTALLS = 5;
```

Comment:
Just made this a const, although I'd rather keep it here since it's only used in this one place